docs/snippets/CMakeLists.txt

```diff
@@ -18,8 +18,10 @@ endif()
 
 file(GLOB SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp"
                   "${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp"
-                  "${CMAKE_CURRENT_SOURCE_DIR}/src/*.c")
-file(GLOB GPU_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/gpu/*.cpp")
+                  "${CMAKE_CURRENT_SOURCE_DIR}/src/*.c"
+                  "${DOCS_SOURCE_DIR}/articles_en/assets/snippets/*.cpp"
+                  "${DOCS_SOURCE_DIR}/articles_en/assets/snippets/*.c")
```

Comment:
looks like we don't have files based on such patterns https://github.com/openvinotoolkit/openvino/actions/runs/10163458768/job/28106404309?pr=25802#step:6:7

In logs we can see that only main.c and main.cpp are built